pypeit/core/flux_calib.py

```diff
@@ -1080,6 +1080,7 @@ def get_mask(wave_star, flux_star, ivar_star, mask_star,
     msgs.info(" Masking Below the atmospheric cutoff")
     atms_cutoff = wave_star <= 3000.0
     mask_bad[atms_cutoff] = False
+    gpm = np.where(mask_bad)
```

Comment:
(A bit nit-picky)  If it makes sense, I'd like to keep the convention that `bpm` and `gpm` are boolean arrays.  Might be easiest to change this to `gpm = mask_bad.copy()`?  Otherwise, we could replace `mask_bad` with `gpm` throughout the function?